### PR TITLE
Update supported browsers

### PR DESCRIPTION
**Pull request by shaal**
While working with Outline on a project, I noticed that `clamp()` in CSS is converted to `max` and `min`.
clamp is supported in all modern browsers - https://caniuse.com/mdn-css_types_clamp

## Description

Current browsers supported (tested by running `npx browserslist`)
```
and_chr 98
and_ff 96
and_qq 10.4
and_uc 12.12
android 98
baidu 7.12
chrome 99
chrome 98
chrome 97
edge 99
edge 98
firefox 97
firefox 96
ios_saf 15.2-15.3
ios_saf 15.0-15.1
ios_saf 14.5-14.8
ios_saf 14.0-14.4
kaios 2.5
safari 15.2-15.3
safari 15.1
safari 15
safari 14.1
safari 14
samsung 16.0
samsung 15.0
```

New proposed supported versions (this PR), based on https://nx.dev/guides/browser-support
```
chrome 99
chrome 98
edge 99
edge 98
firefox 97
firefox 96
firefox 91
ios_saf 15.2-15.3
ios_saf 15.0-15.1
ios_saf 14.5-14.8
ios_saf 14.0-14.4
safari 15.2-15.3
safari 15.1
safari 15
safari 14.1
safari 14
```

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Visual Testing
- [ ] Automated Testing
- [ ] Accessibility Testing

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

<a href="https://gitpod.io/#https://github.com/phase2/outline/pull/340"></a>

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
-not dead, 
-not ie > 0, 
-not opera > 0, 
-not op_mini all, 
-not op_mob > 0,
-
+last 2 Chrome version
+last 2 Firefox version
+last 2 Edge major versions
+last 2 Safari major version
```
Comment (himerus) [inline .browserslistrc]: `version` or `versions`?

File: .browserslistrc [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
+last 2 iOS major versions
+Firefox ESR
```
Comment (himerus) [inline .browserslistrc:9]: Not required IMHO. If our contracts aren't by default using this, we don't want any adverse effects by supporting it.

**Review by himerus — CHANGES_REQUESTED** [review]
What I'm curious about, the original issue, being that `clamp()` was being converted when it shouldn't be:

- It didn't seem like we were supporting IE already (it was IE I was assuming was the culprit)
- I'm curious which of the browsers that were supported by the `browserlist list` command actually were causing this issue.


I AM a bit hesitant to remove the Android and Samsung browsers from the list. I think if we add them in specifically, we'll be good here.

File: .browserslistrc (hunk 2 above)
Comment (himerus) [reply .browserslistrc:9]: I do see this seems to be a standard. Will leave as is.

**Comment by himerus** [comment]
Latest commit updates and results in the following:
```
and_chr 98
and_ff 96
chrome 99
chrome 98
edge 99
edge 98
firefox 97
firefox 96
firefox 91
ios_saf 15.2-15.3
ios_saf 15.0-15.1
ios_saf 14.5-14.8
ios_saf 14.0-14.4
safari 15.2-15.3
safari 15.1
safari 15
safari 14.1
safari 14
samsung 16.0
samsung 15.0
```

This removes essentially `and_qq`, `and_uc`, `baidu` and `kaios`. It appears one of those non-US browsers was causing the issue.

**Review by himerus — APPROVED** [review]

**Review by grshane — APPROVED** [review]
❤️‍🔥

**Pull request merged**